src/ploneintranet/suite/tests/lib/keywords.robot

```diff
@@ -392,6 +439,11 @@ I cannot see the document
     Click link  Documents
     Page should not contain  ${title}
 
+Allan has the option to create a document
+    I am logged in as the user allan_neece
+    I go to the Shareholder Information Workspace
```

Comment:
This is a 'consume' workspace which is the lowest privilege level. That's OK for testing privilege enhancement, but not for reducing access rights. For that you'd need a different workspace with a higher default policy. See other comment.